db/Answer.js

```diff
@@ -10,16 +10,17 @@ module.exports.findAll = function findAll(where = {}) {
     .select();
 };
 
-
-module.exports.findById = function findById(id) {
+module.exports.findById = function findById(id, where = {}) {
```

Comment:
remove the `where` arg (here and elsewhere) as discussed yday - any further filtering can be added at repository layer